frontend/src/components/staff/CreatePackage.jsx

```diff
@@ -252,6 +252,11 @@ export default function CreatePackage({ refresh, onRefresh }) {
 }
 
 CreatePackage.propTypes = {
-  refresh: PropTypes.bool.isRequired,
-  onRefresh: PropTypes.func.isRequired,
+  refresh: PropTypes.bool,
+  onRefresh: PropTypes.func,
+};
+
+CreatePackage.defaultProps = {
+  refresh: false,
+  onRefresh: () => {},
```

Comment:
nit: setting to undefined will guarantee that the component either had a function passed or not. with `() => {}`, it'd be difficult to determine whether it was passed when the component was called or if no function was passed at all
```suggestion
  onRefresh: undefined,
```